README.md

```diff
@@ -199,10 +201,45 @@ declare const Given, When, Then;
 const {given, when, then} = require('cypress-cucumber-preprocessor/resolveStepDefinition')
 ```
 
+## VueJS configuration example
+
+When using with a project using @vue/cli and wanting to use the existing prefix of test/e2e, the following adjustments are necessary:
```

Comment:
you say test/e2e but set up things for tests/e2e, could you fix one or the other please? :) 
Also, is appRoot a vuejs pattern? If not, I think it's understandable to just leave the paths as if going from the root